src/pages/inventory/index.tsx

```diff
@@ -0,0 +1,265 @@
+import React, { useState, useEffect } from 'react';
+import Box from '@mui/material/Box';
+import Table from '@mui/material/Table';
+import TableBody from '@mui/material/TableBody';
+import TableCell from '@mui/material/TableCell';
+import TableContainer from '@mui/material/TableContainer';
+import TableHead from '@mui/material/TableHead';
+import TableRow from '@mui/material/TableRow';
+import TextField from '@mui/material/TextField';
+import MenuItem from '@mui/material/MenuItem';
+import { Button, Menu, Pagination, Typography } from '@mui/material';
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
+import ClearIcon from '@mui/icons-material/Clear';
+import { createClient } from '@supabase/supabase-js';
+import AddIcon from '@mui/icons-material/Add';
+
+type InventoryItem = {
+  id: number;
+  item: string;
+  type: string;
+  quantity: number;
+  category: string;
+}
+
+
+const Inventory = () => {
+
+  const [originalData, setOriginalData] = useState<InventoryItem[]>([]);
+  const [data, setData] = useState<InventoryItem[]>([]);
+  const [type, setType] = useState('');
+  const [category, setCategory] = useState('');
+  const [status, setStatus] = useState('');
+  const [search, setSearch] = useState('');
+  const [anchorType, setAnchorType] = useState<null | HTMLElement>(null);
+  const [anchorCategory, setAnchorCategory] = useState<null | HTMLElement>(null);
+  const [anchorStatus, setAnchorStatus] = useState<null | HTMLElement>(null);
+  const [currentPage, setCurrentPage] = useState(1);
+  const itemsPerPage = 10;
```

Comment:
Do we want to hard-code this or make it depending on page? If we want to do it later, can we add a //TODO?

Reply:
Is this in reference to the itemsPerPage?

Reply:
yes, sorry for not being explicit.